test/integration/deployment/util.go

```diff
@@ -87,6 +94,40 @@ func newDeployment(name, ns string, replicas int32) *v1beta1.Deployment {
 	}
 }
 
+// newDeployment returns a RollingUpdate Deployment with with a fake container image
+func newAppsV1beta2Deployment(name, ns string, replicas int32) *v1beta2.Deployment {
```

Comment:
I have to create a new set of functions for `apps/v1beta2` so that i am able to test both `apps/v1beta2` and `extensions/v1beta1` versions accordingly.